dojo/group/utils.py

```diff
@@ -31,7 +32,7 @@ def group_post_save_handler(sender, **kwargs):
         group.auth_group = auth_group
         group.save()
         user = get_current_user()
-        if user:
+        if user and not(settings.AZUREAD_TENANT_OAUTH2_GET_GROUPS):
             # Add the current user as the owner of the group
```

Comment:
Is there a reason to make the user owner? Is the idea to make sure there is at least 1 owner? Making a user owner allows them to add other members to the group. Although I am not sure what permissions are needed for a user to be able to access the UI elements for this.